### ISSUE TITLE
Feedback/usage collections

### ISSUE THREAD

**Issue by albertwoo** [slaveOftime/Fun.Blazor#70]
Feel free to share what is your usage experience of this library/framework. If you can please also post some screenshots or code examples.

**Comment by masq6r** [comment]
I love the multiple ways to get the components updated, the `adapt` CE in particular. But I had a very hard time reading and understanding the documentation because it's not well organised. For example, the `Get Started` section is not quite entry level, maybe putting a complete walkthrough to build a simple app like counter/todo is better (I know there is a counter snippet, but it's not part of getting started).

**Comment by albertwoo** [comment]
@masq6r thanks for the feedback. I updated the doc to use the fun-wasm as the started template, that one is the simplest one and only with counter demo code.

**Comment by 1eyewonder** [comment]
## Summary

So I attempted to use this library for a side project and did not have any luck after playing around with different approaches for over an hour. Here are my notes to help improve the DX. Know that this project looks really promising and I'm excited to try it again in the future!

## Potential Tooling Improvements

Being that paket is used within the F# community, it may be nice to add some support for this. If you aren't looking to add it that is fine too, but calling it out in the docs would be beneficial.

## Documentation Improvements

I agree with the above sentiment that the documentation is hard to follow. Here are some things I may look to address:

### Organization

- I feel the ordering of the side bar can be improved to help ease in users. i.e. go from "About" to "Getting Started" etc.
- Some of the pages feel like walls of text and could be broken out into separate pages

### Walkthroughs

A walk-through document would be great to see. For example, I am trying to add a Blazor server project to my solution and use MudBlazor as my component library. What do the setup steps looks? As a first time user I expect it to magically work out of the box with no additional steps after installing the template. The below was the view I got from the template and I am unsure if this is what my starting point is supposed to look like:

![Image](https://github.com/user-attachments/assets/06de64a5-cae4-42de-b128-03b1d2e9a94e)

Similar to the above sentiment, additional walk-throughs on the CLI tool would be nice as I don't know if this is needed tool or an additional feature.

### Visualization

I feel like the font size is fairly small and could be up-sized a bit, but this is more of a nitpick since I can always zoom in on my browser.

**Comment by albertwoo** [comment]
@1eyewonder thanks for trying this and give your feedback.
The document needs improvement, I already tried with some contributers. But I am not good at it. Your contribute will be welcome.

**Fun.Blazor** is still just **blazor**, so if you want to use it with your own customization, you should try to check blazor official documents first. Like component lifecycle etc.

In your "Walkthroughs", you mentioned, you do are trying to use MudBlazor with no luck. I think you should first use Fun.Blazor template to get started, then check MudBlazor official document for the necessary steps. Fun.Blazor.MudBlazor is just DSL binding which provide bindings for its componnets. There is no magic out of the box for third party library. There is also a demo which is using MudBlazor + Blazor official style template (Server + WASM): https://github.com/slaveOftime/Fun.Blazor.Samples/tree/main/samples/FunBlazor.MudBlazorDemo
